Assets/MixedRealityToolkit/_Core/Definitions/InteractionDefinition.cs

```diff
@@ -41,6 +59,10 @@ public InteractionDefinition(uint id, AxisType axisType, Devices.DeviceInputType
         /// </summary>
         public InputAction InputAction { get; set; }
 
+        /// <summary>
+        /// Action to be raised to the Input Manager when the input data has changed.
+        /// </summary>
+        public InputAction InputHoldAction { get; set; }
```

Comment:
Should we also include a Pointer action as well?

Originally I had these defined in the Input Action Profile itself.

Reply:
Except that Pointer is unique to pointers ONLY, Hold is a held state for any input

Reply:
not true, it was a global action for all pointers